internal/configs/sharding/mainnet.go

```diff
@@ -17,9 +17,6 @@ const (
 
 	mainnetVdfDifficulty = 50000 // This takes about 100s to finish the vdf
 
-	// TODO: remove it after randomness feature turned on mainnet
-	mainnetRandomnessStartingEpoch = 100000
-
```

Comment:
according to this comment, Does this note mean that the VRF random function has not been officially enabled on the main network until then? That is, the description of the relevant VRF in the white paper has not been implemented yet?

Reply:
Yes, it was an old implementation that's never be launched in mainnet.